packages/react-components/react-theme/src/global/colors.ts

```diff
@@ -848,54 +848,94 @@ const charcoal: ColorVariants = {
   tint60: '#f7f7f7',
 };
 
-export const sharedColors: GlobalSharedColors = {
-  darkRed,
-  burgundy,
-  cranberry,
+export type StatusSharedColors = 'red' | 'green' | 'darkOrange' | 'yellow' | 'berry' | 'lightGreen' | 'marigold';
+
+export const statusSharedColors: Pick<GlobalSharedColors, StatusSharedColors> = {
   red,
+  green,
   darkOrange,
-  bronze,
+  yellow,
+  berry,
+  lightGreen,
+  marigold,
+};
+
+export type PersonaSharedColors =
+  | 'darkRed'
+  | 'cranberry'
+  | 'pumpkin'
+  | 'peach'
+  | 'gold'
+  | 'brass'
+  | 'brown'
+  | 'forest'
+  | 'seafoam'
+  | 'darkGreen'
+  | 'lightTeal'
+  | 'teal'
+  | 'steel'
+  | 'blue'
+  | 'royalBlue'
+  | 'cornflower'
+  | 'navy'
+  | 'lavender'
+  | 'purple'
+  | 'grape'
+  | 'lilac'
+  | 'pink'
+  | 'magenta'
+  | 'plum'
+  | 'beige'
+  | 'mink'
+  | 'platinum'
+  | 'anchor';
+
+export const personaSharedColors: Pick<GlobalSharedColors, PersonaSharedColors> = {
+  darkRed,
+  cranberry,
   pumpkin,
-  orange,
   peach,
-  marigold,
-  yellow,
   gold,
   brass,
   brown,
-  darkBrown,
-  lime,
   forest,
   seafoam,
-  lightGreen,
-  green,
   darkGreen,
   lightTeal,
   teal,
-  darkTeal,
-  cyan,
   steel,
-  lightBlue,
   blue,
   royalBlue,
-  darkBlue,
   cornflower,
   navy,
   lavender,
   purple,
-  darkPurple,
-  orchid,
   grape,
-  berry,
   lilac,
   pink,
-  hotPink,
   magenta,
   plum,
   beige,
   mink,
-  silver,
   platinum,
   anchor,
+};
+
+// These shared colors are currently not used in themes
+// Not exported from the package, we can consider removing them
+export const unusedSharedColors = {
```

Comment:
As a follow up task, the internal structure of the file must be refactored and token pipeline updated.